packages/polaris-viz/src/components/FunnelChart/Chart.tsx

```diff
@@ -209,6 +220,9 @@ export function Chart({
         width={width}
         height={drawableHeight}
         fill={`url(#${gradientId})`}
+        style={{
+          pointerEvents: 'none',
+        }}
```

Comment:
Previously the mask was preventing the Tooltip from being able to register a hover mouse event